rotkehlchen/exchanges/exchange.py

```diff
@@ -417,27 +417,26 @@ def query_income_loss_expense(
             start_ts: Timestamp,
             end_ts: Timestamp,
             only_cache: bool,
-    ) -> list[LedgerAction]:
+    ) -> list['HistoryEvent']:
         """Queries the local DB and the exchange for the income/loss/expense history of the user
 
         If only_cache is true only what is already cached in the DB is returned without
         an actual exchange query.
         """
-        db = DBLedgerActions(self.db, self.db.msg_aggregator)
-        filter_query = LedgerActionsFilterQuery.make(
+        db = DBHistoryEvents(self.db)
+        filter_query = HistoryEventFilterQuery.make(
             from_ts=start_ts,
             to_ts=end_ts,
             location=self.location,
         )
-
         with self.db.conn.read_ctx() as cursor:
             # has_premium True is fine here since the result of this is not user facing atm
-            ledger_actions = db.get_ledger_actions(cursor, filter_query=filter_query, has_premium=True)  # noqa: E501
+            events = db.get_history_events(cursor, filter_query=filter_query, has_premium=True)
             if only_cache:
-                return ledger_actions
+                return events
 
             ranges = DBQueryRanges(self.db)
-            location_string = f'{self.location!s}_ledger_actions_{self.name}'
+            location_string = f'{self.location!s}_history_events_{self.name}'
```

Comment:
is the `!s` needed here?

Reply:
Was before. Explicitness instead of relying on language's quirks I guess.